README.md

```diff
@@ -6,10 +6,14 @@ i currently suck at python, and i'm not really thinking too much, so this code i
 
 ## Installing 
 
-`pip install -r requirments.txt`
+`pip install -r requirements.txt`
```

Comment:
still. You can't just copy-pasta this. Also instructions for the db would be nice...